senior_teacher.rb

```diff
@@ -1,21 +1,18 @@
-class SeniorTeacher
+require_relative 'student'
+require_relative 'highfiveable'
+require_relative 'teacher_do'
+
+class SeniorTeacher < Student
+  include HighFiveable
```

Comment:
This reads as "SeniorTeacher is a Student" which is incorrect. What would be a good name for the Parent class?